warehouse/clickhouse/clickhouse.go

```diff
@@ -270,28 +278,41 @@ func generateArgumentString(arg string, length int) string {
 
 // typecastDataFromType typeCasts string data to the mentioned data type
 func typecastDataFromType(data string, dataType string) interface{} {
+
 	switch dataType {
 	case "int":
 		i, err := strconv.Atoi(data)
 		if err != nil {
+			if disableNullable {
```

Comment:
Maintain a map[string]interface{} where key is dataType and value is the default value for that type.

Let the error be caught after switch and set the value corresponding to that dataType on error.